### PR TITLE
[ticket/14019] Add posting_topic_title_before template event

### PR DESCRIPTION
**Pull request by Zoddo**
Allows to add some information on the left of the topic title in the posting form

https://tracker.phpbb.com/browse/PHPBB3-14019

### CODE REVIEW

**Comment by Zoddo** [comment]
!set Event

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -1052,6 +1052,14 @@ posting_poll_body_options_after
 * Since: 3.1.4-RC1
 * Purpose: Add content after the poll options on creating a poll
 
+posting_topic_title_prepend
```
Comment (marc1706) [inline phpBB/docs/events.md]: posting_topic_title_before

Comment (Zoddo) [inline phpBB/docs/events.md]: [viewtopic_topic_title_prepend](https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/docs/events.md#viewtopic_topic_title_prepend) also use `prepend`.

Comment (marc1706) [inline phpBB/docs/events.md]: To be honest, we never really decided upon using append/prepend or after/before. For non-native English speakers the latter one (after/before) will be easier to understand.

Comment (Zoddo) [inline phpBB/docs/events.md]: renamed

**Comment by marc1706** [comment]
New ticket: https://tracker.phpbb.com/browse/PHPBB3-14085

**Pull request merged**